libs/ConfigKit/include/Config.h

```diff
@@ -1,28 +1,31 @@
 // Leka - LekaOS
-// Copyright 2022 APF France handicap
+// Copyright 2023 APF France handicap
 // SPDX-License-Identifier: Apache-2.0
 
 #pragma once
 
+#include <array>
+#include <cstdint>
 #include <filesystem>
 
 namespace leka {
 
+template <std::size_t SIZE>
```

Comment:
we can make the default `1`

```suggestion
template <std::size_t SIZE = 1>
```